Unit3_Real/Unit3_Real/Controller/StockDetailViewController.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  StockDetailViewController.swift
+//  Unit3_Real
+//
+//  Created by EricM on 9/4/19.
+//  Copyright © 2019 EricM. All rights reserved.
+//
+
+import UIKit
+
+class StockDetailViewController: UIViewController {
+    var apple: Stock!
+    @IBOutlet weak var image2Outlet: UIImageView!
+    @IBOutlet weak var dateLabel: UILabel!
+    @IBOutlet weak var openLabel: UILabel!
+    @IBOutlet weak var closeLabel: UILabel!
+    
+    
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        if apple.open < apple.close{
```

Comment:
This logic should really be done in the model. Make an instance function in the Stocks model that compares the stored properties open and close. You would want to return something, you call that func in the viewDidLoad and based on the results you update the view color and image appropriately.